lib/fathead/flask/parse.py

```diff
@@ -0,0 +1,188 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import os
+from bs4 import BeautifulSoup
+import html
+import unicodedata
+
+URL_ROOT = 'http://flask.pocoo.org/docs/0.12/'
+DOWNLOADED_HTML_PATH = 'download/'
+
+class Data(object):
+    """
+    Class to store parsed data
+    """
+    
+    def __init__(self, file):
+        self.HTML = ""
+        self.FILE = file
+        self.load_data()
+
+    def load_data(self):
+        with open(self.FILE,'r') as data_file:
+            document = data_file.read()
+            self.HTML = document
+
+    def get_raw_data(self):
+        return self.HTML
+
+    def get_file(self):
+        return self.FILE
+
+
+class Parser(object):
+    """
+    Parser to parse HTML files
+    """
+    
+    def __init__(self, data_object):
+        self.file_being_used = data_object.get_file()
+        self.file_name = self.file_being_used.split('/').pop()
+        self.sections = []
+        self.soup = BeautifulSoup(data_object.get_raw_data(), 'html.parser')
+
+    def parse_for_data(self):
+        for header in self.soup.find_all(['dl']):
+            section = self.parse_section(header)
+            section['example'] = self.clean_formatting(self.get_example(header))
+                    
+            if section['paragraph']:
+                section['paragraph'] = section['paragraph'].replace('\n','')
+                if section['example']:
+                    section['abstract'] = '<section class="prog__container><p>{}</p><pre><code>{}</code></pre></section>'.format(section['paragraph'], section['example'])
```

Comment:
@shanmbic you're missing a closing quote (`"`) on the `class="...`, causing the content to not display.

Please add closing quotes and then this should be good for merge.

Please make sure you check to see it's working on Beta 👍